Contrib/CCCS/PreCalculus/2.5/CCD_CCCS_Openstax_AlgTrig_AT-1-001-AS_2_5_32.pg

```diff
@@ -45,6 +45,8 @@ $f=Formula("$a*x^2+$b*x+$c")->reduce;
 
 $ans = ($b)**2-4*$a*$c;
 
+$ans=0;
+
```

Comment:
```suggestion
```
I think you do not want to overwrite the value of $ans here.

Reply:
You are right, did not mean to do that, it is fixed now.